subprojects/common/file-storage/src/main/kotlin/com/avito/filestorage/RemoteStorageFactory.kt

```diff
@@ -0,0 +1,55 @@
+package com.avito.filestorage
+
+import com.avito.http.HttpLogger
+import com.avito.http.RetryInterceptor
+import com.avito.logger.Logger
+import com.avito.logger.LoggerFactory
+import com.avito.logger.create
+import com.avito.time.TimeProvider
+import okhttp3.HttpUrl.Companion.toHttpUrlOrNull
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import java.util.concurrent.TimeUnit
+
+object RemoteStorageFactory {
+
+    private const val TIMEOUT_SEC = 120L
```

Comment:
Make a task for monitoring the time of `file-storage` requests because 120s is a random constant

Reply:
We can do it in MBS-10704